doc/install/build_install.rst

```diff
@@ -706,27 +721,37 @@ then choose GCC 7.1.0
 Note that the C and C++ compiler wrappers are called ``cc`` and ``CC``
 respectively on Cray systems.
 
+.. note::
+
+    When targeting the cuda backend, the compiler has to be compatible with the
+    respective cuda toolkit version. Thus, it may be necessary to switch the
+    compiler. On Piz Daint, for example, one would do the following at the time
+    of this writing:
+
+    .. code-block:: bash
+
+        $ module load daint-gpu/21.09
+        $ module load craype-accel-nvidia60
+        $ module swap cudatoolkit/21.5_11.3 # switch to newer cuda toolkit
+        $ module swap gcc/9.3.0 # switch to 9.3.0 from 11.2.0
+
+
 CMake detects that it is being run in the Cray programming environment, which makes
 our lives a little bit more difficult (CMake sometimes tries a bit too hard to help).
 To get CMake to correctly link our code, we need to set the ``CRAYPE_LINK_TYPE``
 environment variable to ``dynamic``.
 
 .. code-block:: bash
 
-    export CRAYPE_LINK_TYPE=dynamic
+    $ export CRAYPE_LINK_TYPE=dynamic
 
-Putting it all together, a typical workflow to build Arbor on a Cray system is:
+.. note::
 
-.. code-block:: bash
+    This step is no longer required beginning with the Cray PE 10.06 release, where
```

Comment:
If no longer required, then remove.

Reply:
Depends on the Cray PE version. What do you suggest instead?

Reply:
"Cray PE" is not defined, so it may not be obvious that it refers to our how a user controls it. I image it's an env with a set of GCC builds etc. one can load? Or is it done system wide and has the user no choice?

Reply:
Cray PE= Cray Programming Environment. Yes it comes with a bunch of tools: compilers, MPI, scientific libraries, but it also sets a bunch of environment variables etc. It usually the minimum you have on a system provided by Cray (HPE).